gcloud/test__helpers.py

```diff
@@ -221,6 +222,15 @@ def test_read_from_cli_info(self):
         project_id = self.callFUT('test-project-id')
         self.assertEqual('test-project-id', project_id)
 
+    def test_gae_without_expanduser(self):
+        import sys
+        try:
+            sys.modules['pwd'] = None  # Blocks pwd from being imported.
```

Comment:
I guess we don't care about putting back the "old" module if it was already imported.